docs/docs-content/tutorials/getting-started/palette-edge/edge-cluster-profile.md

```diff
@@ -0,0 +1,159 @@
+- A [Palette account](https://www.spectrocloud.com/get-started) with
+  [tenant admin](../../../tenant-settings/tenant-settings.md) access.
+- One available IP address on the same network as the Edge host for the MetalLB load balancer.
+
+## Create Cluster Profile
+
+Log in to [Palette](https://console.spectrocloud.com/) and select **Profiles** from the left main menu. Click **Add
+Cluster Profile** to create your cluster profile.
+
+Follow the wizard to create a new profile. In the **Basic Information** section, assign `gs-profile` as the name, and
+provide a brief profile description. Set the type as **Full** and add the tag `env:edge`. You may leave the version
+empty, but note that the version defaults to `1.0.0` if not specified. Click **Next** to continue.
+
+The **Cloud Type** section allows you to choose the infrastructure provider for the cluster. Select **Edge Native** and
+click **Next**.
+
+The **Profile Layers** section defines the packs that compose the profile.
+
+Add the **BYOS Edge OS** pack to the OS layer. This pack enables you to use the custom image you built in the
+[Build Edge Artifacts](./build-edge-artifacts.md) tutorial as the operating system for the cluster nodes.
+
+| **Pack Name** | **Version** | **Registry**     | **Layer**        |
+| ------------- | ----------- | ---------------- | ---------------- |
+| BYOS Edge OS  | 2.1.0       | Palette Registry | Operating System |
+
+Under **Pack Details**, select **Values** to open the YAML editor. Replace the default layer manifest with the custom
+manifest generated in the [Build Edge Artifacts](./build-edge-artifacts.md) tutorial. This makes the cluster pull the
+provider images from the image registry specified in the [Build Edge Artifacts](./build-edge-artifacts.md) tutorial
+during deployment.
+
+The following image displays the OS layer with the custom manifest and registry credentials.
+
+![A screenshot of the cluster profile creation step with the OS layer.](/getting-started/getting-started_introduction-edge_edge-cluster-profile_byos-cluster-profile.webp)
+
+Click **Next Layer** to continue. Add the following Kubernetes layer to your cluster profile. Ensure the Kubernetes
+version matches the version used in the provider images.
+
+| **Pack Name**         | **Version** | **Registry**     | **Layer**  |
+| --------------------- | ----------- | ---------------- | ---------- |
+| Palette Optimized K3s | 1.32.3      | Palette Registry | Kubernetes |
+
+Under **Pack Details**, select **Values** and replace the predefined `cluster-cidr` and `service-cidr` IP CIDRs if they
+overlap with the host network. For example, you can set `cluster-cidr` to `"100.64.0.0/18"` and `service-cidr` to
+`"100.64.64.0/18"`. This prevents any routing conflicts in the internal pod networking.
+
+![A screenshot of the cluster profile creation step with the Kubernetes layer.](/getting-started/getting-started_introduction-edge_edge-cluster-profile_cluster-profile-k8s.webp)
+
```

Comment:
I think it may be worth it to mention why there is not a CSI: Many Edge deployments are small and the Edge hosts don't have storage capacity so we don't require it as part of infra layer. But if their application needs persistent storage they probably need to add a CSI.